### PR TITLE
Release/v0.3.0

### PR DESCRIPTION
**Pull request by maxvisser**
Changelog:
- Update identity email api change https://github.com/teamforus/me/issues/406
- Switching organization on doing transaction https://github.com/teamforus/me/issues/193
- error handling improvements https://github.com/teamforus/me/issues/411 https://github.com/teamforus/me/issues/267
- No internet error https://github.com/teamforus/me/issues/259
- On session destroy: give user a popup notice to logout me app https://github.com/teamforus/me/issues/388
- Log me in flow updates https://github.com/teamforus/me/issues/396
- Add records page as hidden feature https://github.com/teamforus/me/issues/383
- Test transaction implementation https://github.com/teamforus/me/issues/343
- Move logout button on set passcode screen https://github.com/teamforus/me/issues/260
- Implement voucher token (automatically login on webshop) voucher info button https://github.com/teamforus/me/issues/379
- Replace fallback voucher info url https://github.com/teamforus/me/issues/410
- better success modal https://github.com/teamforus/me/issues/377

### CODE REVIEW

**Comment by maestrovsmaster** [comment]
> ## Description
> Can't build an APK with Android Studio.
> When building I will get this error about a string.
> 
> ### Error
> Screen Shot 2020-04-28 at 13 38 23

fixed

File: data/src/main/java/io/forus/me/android/data/net/vouchers/VouchersService.kt [hunk 1]
```diff
@@ -37,4 +36,8 @@ interface VouchersService {
     @POST("api/v1/platform/vouchers/{address}/send-email")
     fun sendEmail(@Path("address") address: String): Observable<Void>
 
+//platform/demo/transactions/  //body = ["state" : "accepted"] , testToken from QR
```
Comment (maxvisser) [inline data/src/main/java/io/forus/me/android/data/net/vouchers/VouchersService.kt:39]: can be removed

File: presentation/src/main/java/io/forus/me/android/presentation/deeplinks/UriToIntentMapper.kt [hunk 2]
```diff
@@ -27,15 +28,18 @@ class UriToIntentMapper(private val mContext: Context, private val navigator: Na
     private fun mapAppLink(uri: Uri): Intent? {
         val host = uri.host!!.toLowerCase()
 
+        Log.d("forus", "host=$host") // host=identity-restore
```
Comment (maxvisser) [inline presentation/src/main/java/io/forus/me/android/presentation/deeplinks/UriToIntentMapper.kt:31]: can be removed

File: presentation/src/main/java/io/forus/me/android/presentation/view/base/lr/LRPanelImpl.kt [hunk 3]
```diff
@@ -59,6 +62,13 @@ class LRPanelImpl @JvmOverloads constructor(context: Context, attrs: AttributeSe
         if (vs.loading) {
             LR_ViewAnimator.showIfNotYet(LAYER_LOADING)
         } else if (vs.loadingError != null) {
+
+            Log.d("networkError","LRPanelImpl networkError "+vs.loadingError!!::class)
```
Comment (maxvisser) [inline presentation/src/main/java/io/forus/me/android/presentation/view/base/lr/LRPanelImpl.kt:66]: can be removed

**Comment by maxvisser** [comment]
added a UI fix and a fix for scanning when there is 0 euro's on the voucher

**Comment by GerbenBosschieter** [comment]
I am getting a crash when trying to connect from other device.
This only happend on FixUI branch. But after merge happends to this branch also.

Not sure if this is a bu or just a build/phone problem.

@jknnr @maxvisser could you reproduce?

**Comment by maxvisser** [comment]
error i got:
```
E/AndroidRuntime: FATAL EXCEPTION: main
    Process: io.forus.me.dev, PID: 32617
    android.view.InflateException: Binary XML file line #48: ScrollView can host only one direct child
    Caused by: java.lang.IllegalStateException: ScrollView can host only one direct child
```

**Comment by GerbenBosschieter** [comment]
> added a UI fix and a fix for scanning when there is 0 euro's on the voucher

This bug is still present in this branch.

**Comment by jknnr** [comment]
> > added a UI fix and a fix for scanning when there is 0 euro's on the voucher
> 
> This bug is still present in this branch.

could you test the latest PR on dev for this thing + crash on the pairing screen?

**Pull request merged**